adyenofficial.php

```diff
@@ -185,11 +185,13 @@ public function __construct()
     }
 
     /**
+     * @param $mode
+     *
      * @return string[]
      */
-    public static function getAdyenConfigNames()
```

Comment:
If I'm not mistaken this is a full list of all the configs and is used also when the plugin is being uninstalled. In case we do this change then based on the config live - test it will only remove part of the configurations from the db instead of all

Reply:
https://github.com/Adyen/adyen-prestashop/pull/203/commits/74365fa57356f72033997b7a74ea5128e7e68ec4